frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/OnboardingModal.tsx

```diff
@@ -129,6 +129,7 @@ export function OnboardingModal(props: IOnboardingModalProps) {
                 }}
                 style={{
                   marginBottom: '0.5em',
+                  paddingRight: '5em',
```

Comment:
@o-farag Because you are only changing a horizontal size, I'm guessing this works because of the length of the English string. Please test in other languages - this fix may not be effective there.

Reply:
Oh, good catch